release notes/v0.52.0.md

```diff
@@ -0,0 +1,141 @@
+- [browser#1326](https://github.com/grafana/xk6-browser/pull/1326) Adds forgotten `BrowserContext.browser` and `Page.context` mappings.
+- [browser#1360](https://github.com/grafana/xk6-browser/pull/1360), [browser#1327](https://github.com/grafana/xk6-browser/pull/1327), [browser#1335](https://github.com/grafana/xk6-browser/pull/1335), [browser#1365](https://github.com/grafana/xk6-browser/pull/1365), [browser#1313](https://github.com/grafana/xk6-browser/pull/1313), [browser#1322](https://github.com/grafana/xk6-browser/pull/1322), [browser#1330](https://github.com/grafana/xk6-browser/pull/1330), [browser#1343](https://github.com/grafana/xk6-browser/pull/1343), [browser#1345](https://github.com/grafana/xk6-browser/pull/1345), [browser#1352](https://github.com/grafana/xk6-browser/pull/1352) Turns the `Browser`, `BrowserContext`, `ElementHandle`, `JSHandle`, `Keyboard`, `Mouse`, `Locator`, and `Page` types' panics into errors for stability and better error handling.
+
+## Bug fixes
+
+- [#3774](https://github.com/grafana/k6/pull/3774) fixes a `require` warning for those tests using the stdin.
+- [#3776](https://github.com/grafana/k6/pull/3776) fixes a panic caused by passing an undefined handler to timers.
+- [#3779](https://github.com/grafana/k6/pull/3779) fixes a panic caused by registering an undefined handler in gRPC streams.
+- [xk6-websockets#73](https://github.com/grafana/xk6-websockets/pull/73) fixes a panic caused by registering an undefined handler in WebSockets.
+- [browser#1369](https://github.com/grafana/xk6-browser/pull/1369) Improves `valueFromRemoteObject` `null` detection by returning a Go `nil` instead of `"null"` as a `string`.
+- [browser#1386](https://github.com/grafana/xk6-browser/pull/1386) Correctly handles empty string flags that don't have a value.
+- [browser#1380](https://github.com/grafana/xk6-browser/pull/1380) Ensures that `JSHandle.evaluate` and `JSHandle.evaluateHandle` both set themselves as the first argument.
+- [browser#1346](https://github.com/grafana/xk6-browser/pull/1346) Fixes an IFrame panic ("we either navigate top level or have old version of the navigated frame") that happens during navigation.
+- [browser#1349](https://github.com/grafana/xk6-browser/pull/1349), [browser#1354](https://github.com/grafana/xk6-browser/pull/1354) Fixes `Request` mappings.
+- [browser#1334](https://github.com/grafana/xk6-browser/pull/1334) Fixes an issue where clicking on a link that opens a new tab never navigates to the href link.
+- [browser#1318](https://github.com/grafana/xk6-browser/pull/1318) Fixes the `Mouse.move` to correctly dispatch a `down` event.
+- [browser#1301](https://github.com/grafana/xk6-browser/pull/1301) Fixes an error that occurs when working with a second tab and navigating to a URL.
+- [browser#1387](https://github.com/grafana/xk6-browser/pull/1387) Fixes a panic when the new document or request is missing.
```

Comment:
```suggestion
- [browser#1369](https://github.com/grafana/xk6-browser/pull/1369) improves `valueFromRemoteObject` `null` detection by returning a Go `nil` instead of `"null"` as a `string`.
- [browser#1386](https://github.com/grafana/xk6-browser/pull/1386) correctly handles empty string flags that don't have a value.
- [browser#1380](https://github.com/grafana/xk6-browser/pull/1380) ensures that `JSHandle.evaluate` and `JSHandle.evaluateHandle` both set themselves as the first argument.
- [browser#1346](https://github.com/grafana/xk6-browser/pull/1346) fixes an IFrame panic ("we either navigate top level or have old version of the navigated frame") that happens during navigation.
- [browser#1349](https://github.com/grafana/xk6-browser/pull/1349), [browser#1354](https://github.com/grafana/xk6-browser/pull/1354) fixes `Request` mappings.
- [browser#1334](https://github.com/grafana/xk6-browser/pull/1334) fixes an issue where clicking on a link that opens a new tab never navigates to the href link.
- [browser#1318](https://github.com/grafana/xk6-browser/pull/1318) fixes the `Mouse.move` to correctly dispatch a `down` event.
- [browser#1301](https://github.com/grafana/xk6-browser/pull/1301) fixes an error that occurs when working with a second tab and navigating to a URL.
- [browser#1387](https://github.com/grafana/xk6-browser/pull/1387) fixes a panic when the new document or request is missing.
```